homeassistant/const.py

```diff
@@ -138,6 +138,7 @@
 CONF_SENDER = 'sender'
 CONF_SENSOR_CLASS = 'sensor_class'
 CONF_SENSORS = 'sensors'
+CONF_SERVER_LOC = 'server_location'
```

Comment:
Do you mean directly into microsoft_face.py? I didn't know that was an option. Will implement hopefully tomorrow.